extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -2,10 +2,17 @@ amp-story-shopping-attachment {
   height: 100% !important;
   width: 100% !important;
   font-family: 'Poppins', sans-serif !important;
-  font-size: calc(var(--story-page-vmin) * 3) !important; /* override with ems for pixel perfect layout */
+  font-size: calc(var(--story-page-vmin) * 3) !important; /* Override with ems for pixel perfect layout. */
   --i-amphtml-shopping-attachment-horizontal-spacing: 1.5em !important;
   --i-amphtml-shopping-attachment-vertical-spacing: 1.2em !important;
   --i-amphtml-shopping-three-percent-overlay: rgba(0, 0, 0,  .03) !important;
+  --i-amphtml-amp-story-shopping-pdp-carousel-height: calc(var(--story-page-vh) * 80 - var(--story-page-vw) * 60) !important;
```

Comment:
Nit: let's stay consistent with the other variables and not add `-amp-story- in the var name